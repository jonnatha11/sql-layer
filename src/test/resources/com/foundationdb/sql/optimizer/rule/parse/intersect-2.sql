SELECT name FROM parent
INTERSECT ALL
SELECT name FROM customers
