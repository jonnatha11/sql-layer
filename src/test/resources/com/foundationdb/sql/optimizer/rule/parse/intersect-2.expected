SelectQuery@2c95f299
  ResultSet@286a31ad[name]
    Project@53e36995[PROJECT[0]]
      Intersect@7728e039(ALL)
        ResultSet@1c98fb4d[name]
          Project@7f9a8ae5[parent.name]
            Select@7b1501d5[]
              TableSource@202b8614(parent)
        ResultSet@96f9b8c[name]
          Project@7e4d092a[customers.name]
            Select@7394e52d[]
              TableSource@345228a3(customers)